spec/support/webmock.rb

```diff
@@ -0,0 +1,8 @@
+require 'rspec'
+require 'webmock/rspec'
+
+RSpec.configure do |config|
+  config.before(:each) { WebMock.enable! }
+  config.after(:suite) { WebMock.disable! }
```

Comment:
This is because of the code coverage - after the suite runs, an upload to Code Climate will happen from within the process.